pipelines/pipelines/tensorflow/prediction/queries/ingest.sql

```diff
@@ -4,24 +4,31 @@
 -- you may not use this file except in compliance with the License.
 -- You may obtain a copy of the License at
 
---     https://www.apache.org/licenses/LICENSE-2.0
+--      http://www.apache.org/licenses/LICENSE-2.0
 
 -- Unless required by applicable law or agreed to in writing, software
 -- distributed under the License is distributed on an "AS IS" BASIS,
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
-with filtered_data as (
+-- Treat "filter_start_value" as the current time, unless it is empty then use CURRENT_DATETIME() instead
```

Comment:
have you looked into using https://cloud.google.com/bigquery/docs/reference/standard-sql/procedural-language#declare instead?

Reply:
unfortunately doesn't work - the BigQuery SDK function we are using only accepts a single SQL statement, not a script